extmod/modasyncio.c

```diff
@@ -255,6 +387,15 @@ STATIC void task_attr(mp_obj_t self_in, qstr attr, mp_obj_t *dest) {
     }
 }
 
+STATIC mp_obj_t task_unary_op(mp_unary_op_t op, mp_obj_t o_in) {
```

Comment:
Hashing of Task should actually work, since commit eaccaa36771f784dfb458b8ba3591f11513824e6.  So I think this bit of code can be removed.

Reply:
I'll confirm it without this, it's very likely that's the case!

Reply:
Yup, seems fine with that patch, awesome.